### PR TITLE
Fix: install needed dependencies to use Hive in Docker image

### LINKED ISSUE #1076: NameError: global name 'hive' is not defined
### Issue Summary
I configure a Hive dataSource and try to execute a simple query 
and i got this error : 
redash_1       | [2016-05-25 14:52:33,854: ERROR/Worker-27] global name 'hive' is not defined
redash_1       | Traceback (most recent call last):
redash_1       |   File "/opt/redash/current/redash/query_runner/hive_ds.py", line 94, in run_query
redash_1       |     connection = hive.connect(**self.configuration.to_dict())
redash_1       | NameError: global name 'hive' is not defined
redash_1       | [2016-05-25 14:52:33,855: ERROR/Worker-27] redash.tasks.refresh_schemas[aae957b8-1c51-428b-8f56-9a058be6cdc9]: Failed refreshing schema for the data source: My Cluster 
redash_1       | Traceback (most recent call last):
redash_1       |   File "/opt/redash/current/redash/tasks/queries.py", line 349, in refresh_schemas

### Technical details:
* Redash Version: latest
* Browser/OS: docker 
* How did you install Redash: with the latest docker-compose-example.yml

### PR DESCRIPTION
**Pull request by nabilblk**
addtional dependencies for Hive Datasource

Fixes #1076.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -40,3 +40,6 @@ python-simple-hipchat==0.4.0
 xlsxwriter==0.8.4
 pystache==0.5.4
 parsedatetime==2.1
+sasl>=0.1.3
+thrift>=0.8.0
+thrift_sasl>=0.1.0
```
Comment (arikfr) [inline requirements.txt]: We keep all the data sources specific requirements in `requirements_all_ds.txt` file. Please move these 3 there as well.

Comment (arikfr) [inline requirements.txt]: Ping.

Comment (nabilblk) [inline requirements.txt]: Sorry for the delay , i will push my change in hours

**Comment by nabilblk** [comment]
hey @arikfr What's the meaning of the tag "Needs More Work" ?

**Comment by arikfr** [comment]
@nabilblk that I'm waiting for changes. For some reason when I checked the pull request it didn't show your updated commit.

Merged now. Thanks!

**Pull request merged**